search/routes/api/serialize.py

```diff
@@ -43,7 +46,9 @@ def _transform_format(cls, fmt: str, paper_id: str, version: int) -> dict:
         }
 
     @classmethod
-    def _transform_latest(cls, document: Document) -> dict:
+    def _transform_latest(cls, document: Document) -> Optional[dict]:
+        if not document.latest:
+            return None
         return {
             "paper_id": document.latest,
             "href": url_for("api.paper", paper_id=document.paper_id,
```

Comment:
This should probably be a canonical URL for the paper ID.

Reply:
Added ``canonical``